### PR TITLE
Clarify consideration of yarn.lock files of dependencies

### PR DESCRIPTION
**Pull request by MajorBreakfast**
Implements #215

### CODE REVIEW

**Comment by MajorBreakfast** [comment]
@thejameskyle I hope you like the phrasing

File: en/docs/yarn-lock.md [hunk 1]
```diff
@@ -48,3 +48,7 @@ break something.
 All `yarn.lock` files should be checked into source control (e.g. git or
 mercurial). This allows Yarn to install the same exact dependency tree across
 all machines, whether it be your coworker's laptop or a CI server.
+
+### Current package only <a class="toc" id="toc-current-package-only" href="#toc-current-package-only"></a>
+
+The `yarn.lock` files of dependencies are disregarded. Only the top level `yarn.lock` file next to the `package.json` of the current package is taken into account.
```
Comment (thejameskyle) [inline en/docs/yarn-lock.md]: Can you put it at the bottom of the top section and say something along the lines of:

> During install Yarn will only use the top-level `yarn.lock` and will ignore any `yarn.lock` files that exist within dependencies. The top-level `yarn.lock` files include everything Yarn needs for the entire dependency tree so all your versions will still be locked.

**Comment by MajorBreakfast** [comment]
I took your sentence and rephrased it a bit. Also I moved it one up (I hope that is what you meant with "top section")

**Comment by MajorBreakfast** [comment]
👍

**Pull request merged**